Dockerfile

```diff
@@ -1,6 +1,46 @@
+## Buildstage ##
+FROM lsiobase/ubuntu:xenial as buildstage
+
+# Build arguments
+ARG BUILD_DATE
+ARG VCS_REF
+ARG VERSION
+ARG DOCKERHUB
+ARG BASEIMAGE
+ARG MODNAME
+
+# Build-time metadata as defined at http://label-schema.org
+LABEL org.label-schema.name="${DOCKERHUB}:${BASEIMAGE}-${MODNAME}" \
```

Comment:
Do these labels appear in the final image? I've never tested this scenario but I would imagine they would be discarded/lost along with the buildstage

Reply:
You're right. Here's my earlier test build [here](https://microbadger.com/images/thecaptain989/striptracks-mod).  I'll remove them.

Reply:
I simplified the Dockerfile and .travis.yml files, removing unused variables, accordingly.